builder-base/Makefile

```diff
@@ -5,6 +5,12 @@ SHELL=bash
 .SUFFIXES:
 
 BASE_DIRECTORY=$(shell git rev-parse --show-toplevel)
+
+# In a codebuild environment the source is not a git repo, but simply a directory.
+ifdef CODEBUILD_SRC_DIR
```

Comment:
why isnt a git repo?  Im just trying to understand the diff of the setup between this and eks-a-build-tooling?

Reply:
took a look at this -- looks like I can tweak a setting, "full clone mode", that could allow the source repo to be a full git clone rather than the default, which is just a directory structure unpacked from a tarball. I'll look at updating the sources to support that

Reply:
ok yeah it's a bit more than a tweak -- the github source for codebuild doesn't support 'full clone mode' out of the box and it's going to require some modifications. Given how much we use the git metadata to orient ourselves in the repositoriy etc I'm going to take the time to set this up properly to pull from git rather than just copy over the source files in a tarball without the git metadata

Reply:
fixed -- as discussed, we'll just run a `git init` in the buildspec for now to initialize the directory structure.